### ISSUE TITLE
CSS for displaying resources

### ISSUE THREAD

**Issue by cjfreeze** [wintermeyer/vutuv#216]
We need generic pages for displaying specific categories of user information.

A resource is a piece of information a user can have multiple of, such as phone numbers. Resources will have several actions, and will require several pages. The first page is index, which displays a list of resources with their information trimmed. For example, the index of addresses may only display the city/state and description. For smaller resources like phone numbers, they will be able to display all of the information. Here is an example of the phoenix default index page for a phone number resource, with fields value and type.
![index](https://cloud.githubusercontent.com/assets/11078691/19899750/304ad4da-a02f-11e6-8f69-2a6c7c5b9d36.png)

As you can see, resources each have a `show`, `edit`, and `delete` button. There is also a `new` link.

Show links to a "more info" page, which would display more complete information about that specific resource. For example, addresses, which cannot fit all of their info in the index page, will display all of their info on the show page. Show pages will also feature an edit and delete button. Here is an example of a default phoenix show page.

![show](https://cloud.githubusercontent.com/assets/11078691/19899787/536888c2-a02f-11e6-9091-704ec0b3dce5.png)

This example has an `edit` link which links to an edit page, and a `back` link which links back to the `index` page.
Note that while the above example does not have a `delete` link, we would like one.

Edit links to a form, which you have already provided basic css for, but feel free to create different form css for resources if you'd like. Here is an example of phoenix's basic edit page.

![form](https://cloud.githubusercontent.com/assets/11078691/19899804/64bbf2d0-a02f-11e6-9295-3989afcbbd43.png)

To summarize, the pages required for any resource are an `index`, a `show`, and an `edit`/`new`, which is just a form. While they don't need to look the same as the examples above, they should at least have everything the above has.

I tried jerry-rigging the user's profile html and css into a resource display page, but it did not translate well. This is most likely because a resource display page is inherently much more simple than a profile page. Please reply to this issue if you have any questions.

**Comment by wintermeyer** [comment]
Just to make sure that we all are on the same page. The following part of the above examples would be replaced by the navigation bar:
![bildschirmfoto 2016-11-01 um 19 08 31](https://cloud.githubusercontent.com/assets/69770/19901234/b7e95286-a066-11e6-8c59-7dfd8060b2a0.png)

And I'd love to have that navigation bar some sort of breadcrumbs.

Examples:
Stefan Wintermeyer / Phone numbers / Home
Stefan Wintermeyer / Addresses

Each breadcrumb would be linked to that page. So in case I'm on a specific phone number I could go back to the complete User information by just clicking on the "Stefan Wintermeyer" link.

**Comment by cfroerup** [comment]
@wintermeyer: Could your comment be in a separate ticket? I don't see it as a part of the original ticket?

**Comment by cfroerup** [comment]
@cjfreeze: For editing we have the page: http://vutuv.dev.freyventures.com/dist/edit.php
We will look into the resource pages

**Comment by wintermeyer** [comment]
I opened a new issue for the breadcrumbs. https://github.com/vutuv/vutuv/issues/218
